core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -357,6 +357,13 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Returns the list of {@link ServerPort}s set so far.
+     */
+    public List<ServerPort> ports() {
+        return ImmutableList.copyOf(ports);
+    }
+
```

Comment:
Could we hide this from the public API? :thinking: We could access it reflectively at worst.

Reply:
Used reflection to bring it. 😄